source/minio-cli/minio-mc/mc-tag.rst

```diff
@@ -0,0 +1,223 @@
+==========
```

Comment:
@poornas @vadmeste added mc tag as I was missing it before, along with versioning functionality.